datadog_checks_dev/datadog_checks/dev/tooling/templates/integration/check/{check_name}/README.md

```diff
@@ -2,7 +2,12 @@
 
 ## Overview
 
-This check monitors [{integration_name}][1] through the Datadog Agent.
+This check monitors [{integration_name}][1] through the Datadog Agent. 
```

Comment:
If this is for all tiles, I don't think this sentence applies. Perhaps we should revise.

Reply:
This isn't this is for the agent check readme